### DISCUSSION TITLE
Animations stops working after update to Tailwind 2.2.2

### DISCUSSION THREAD

**Discussion by akshay-sood** [tailwindlabs/tailwindcss#4752] [Help]
Hi guys,
May be I am missing something, but I would appreciate some help here.
I have a working tailwind css project with Nuxt JS, I have a custom keyframe and animation in tailwind config. And it works fine in 2.1

But as soon as I upgrade to 2.2.2, it stops working. Even any inbuild animation like animate-spin, animate-ping etc. does not work. I change my pacakgae.json to have tailwind 2.1, it starts working again.

My Tailwind Config:

`module.exports = {
  mode: process.env.NODE_ENV ? 'jit' : undefined,
  // mode: 'jit',
  purge: [
    './components/**/*.{vue,js}',
    './layouts/**/*.vue',
    './pages/**/*.vue',
    './plugins/**/*.{js,ts}',
    './nuxt.config.{js,ts}',
    './stub.html'
  ],
  darkMode: false, // or 'media' or 'class'
  theme: {
    extend: {
      colors: {
        accent: {
          light: '#25D461',
          DEFAULT: '#029945',
          dark: '#00682E',
          darker: '#035A21'
        },
        gray: {
          light: '#F5F5F5',
          dark: '#6E6E6E'
        },
        black: {
          DEFAULT: '#343434'
        },
        white: {
          DEFAULT: '#FFFFFF'
        }
      },
      fontFamily: {
        // serif: ['"Montserrat"', 'sans-serif'],
        sans: ['"Montserrat"', 'sans-serif']
        // cursive: ['Montserrat', 'sans-serif']
      },
      screens: {
        'true-hover': { raw: '(hover: hover)' }
      },
      keyframes: {
        'travel-x': {
          '0%': { opacity: '1', transform: 'translateX(0px)' },
          '25%': { opacity: '0', transform: 'translateX(20px)' },
          '26%': { opacity: '0', transform: 'translateX(-20px)' },
          '55%': { opacity: '1', transform: 'translateX(0px)' }
        }
      },
      animation: {
        'travel-x': 'travel-x 1.4s linear infinite'
      }
    }
  },
  variants: {
    extend: {}
  },
  plugins: [
    require('@tailwindcss/forms'),
    require('@tailwindcss/typography')
  ]
}
`

Where I am trying to use it:

`<IconArrow class="h-6 w-6 stroke-current stroke-1 arrow hover:animate-travel-x " />`

I am at a complete loss what's happening here. Is this a  bug, or am I doing something wrong?

**Comment by akshay-sood** [comment]
So I guess the issue is not interesting enough for anyone to respond to?

**Comment by simonswiss** [comment]
Hey!

Are you able to share a GitHub repo that reproduces this issue?

I copied your config file in a new project (simple HTML page) and the animations work fine (including your custom `travel-x` animation added to the config.

Would love to help further, but it's really hard to tell without seeing the actual code and playing with it. I can imagine it's potentially an issue with the paths you have in your `purge` array?
